.github/ISSUE_TEMPLATE/1_activation.md

```diff
@@ -17,34 +17,45 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
 
-<!-- Please tell us exactly how to reproduce the problem you are running into. -->
+<!--
+     Please describe in detail the problem you are running into.
+
+     If you have concrete steps to reproduce the problem, list them:
+
+        1. ...
+        2. ...
+        3. ...
+-->
 
-1. ...
-2. ...
-3. ...
 
-## Logs
 
 <details>
 <summary>Logs</summary>
 
 <!--
-      Include the full logs of the commands you are running between the lines
-      with the backticks below. If you are running any "flutter" commands,
-      please include the output of running them with "--verbose"; for example,
-      the output of running "flutter --verbose create foo".
+     Include the full logs of the commands you are running between the lines
+     with the backticks below. If you are running any "flutter" commands,
+     please include the output of running them with "-v"; for example,
+     the output of running "flutter -v create foo".
```

Comment:
why not --verbose?

Reply:
to line this up with other commands that suggest using `-v`. and why to ask verbosily type `--verbose`?

Reply:
mostly the advantage of saying --verbose is that it's self-documenting. But I don't feel strongly about it.